test/unit/test_assembly.py

```diff
@@ -312,6 +312,18 @@ def test_multi_overlap(self):
             str(Bio.SeqIO.read(outFasta, 'fasta').seq),
             str(Bio.SeqIO.read(expected, 'fasta').seq))
 
+class TestGap2Seq(TestCaseWithTmp):
+    '''Test gap-filling tool Gap2Seq'''
+
+    def test_gapfill(self):
+        from os.path import join
+        inDir = util.file.get_test_input_path()
+        in_scaffold = join(inDir, 'TestOrderAndOrient', 'expected.ebov.doublehit.fasta')
+        with util.file.tempfname(suffix='.filled.fasta') as filled:
+            assembly.gapfill_gap2seq(in_scaffold=in_scaffold,
+                                     inBam=join(inDir, 'G5012.3.testreads.bam'), out_scaffold=filled)
+            self.assertEqualContents(filled, join(inDir, 'TestGap2Seq', 'expected.ebov.doublehit.gapfill.fasta'))
```

Comment:
I agree about it being a small file, but could we achieve the same thing by having the assertion raise with an appropriate message on failure? It would mean changing `assert_md5_equal_to_line_in_file` to have an optional argument (maybe called `failure_message`) that would be passed as the third argument of the [`assertEqual()` comparison it uses](https://github.com/broadinstitute/viral-ngs/blob/master/test/__init__.py#L74). 

Or do you mean it would be helpful to have the expected output in case the developer running `pytest` wants to retain the temp output file for manual inspection and comparison to the expected output?